packages/dev/core/src/Rendering/subSurfaceConfiguration.ts

```diff
@@ -89,6 +89,12 @@ export class SubSurfaceConfiguration implements PrePassEffectConfiguration {
         Constants.PREPASS_IRRADIANCE_TEXTURE_TYPE,
     ];
 
+    /**
+     * The clear color of the render targets. If not provided, defaults to (0, 0, 0, 0)

```

Comment:
Shouldn't it be

```suggestion
     * The clear color of the render targets. If not provided, defaults to (0, 0, 0, 1)
```

Or did you mean if the member is `undefined`?

Reply:
Yes, it is the value if "undefined". I removed the "If not provided" part to make it clearer.